src/components/TitleCardList.tsx

```diff
@@ -152,52 +40,171 @@ const TitleCardList: React.FC<IProps> = ({
   icon,
   isCode,
 }) => (
-  <Table className={className}>
-    <TableHeader>
-      {icon && <StyledIcon name={icon} />}
+  <Box
+    bg="background"
+    boxShadow="tableBoxShadow"
+    width="100%"
+    margin="2rem"
+    border-radius="2px"
+    className={className}
+  >
+    <Flex
+      bg="ednBackground"
+      align="center"
+      justifyContent="space-between"
+      padding="1rem"
+      flexDirection="row-reverse"
+      fontWeight="600"
+      borderBottom="1px"
+      borderBottomStyle="solid"
+      borderBottomColor="text"
+    >
+      {icon && (
+        <ChakraIcon
+          as={Icon}
+          name={icon}
+          _hover={{
+            path: { fill: "tranparent" },
+          }}
+        />
+      )}
       <Translation id={headerKey} />
       {isCode && (
-        <CodeBoxHeader>
-          <Red />
-          <Yellow />
-          <Green />
-        </CodeBoxHeader>
+        <Hide below="s">
+          <Flex>
+            <Box
+              width="12px"
+              height="12px"
+              bg="fail300"
+              marginRight="0.5rem"
+              borderRadius="64px"
+            />
+            <Box
+              width="12px"
+              height="12px"
+              bg="gridYellow"
+              marginRight="0.5rem"
+              borderRadius="64px"
+            />
+            <Box
+              width="12px"
+              height="12px"
+              bg="success300"
+              marginRight="0.5rem"
+              borderRadius="64px"
+            />
+          </Flex>
+        </Hide>
       )}
-    </TableHeader>
+    </Flex>
     {content.map((listItem, idx) => {
-      const { title, description, caption, link, image, alt, id } = listItem
+      let { title, description, caption, link, image, alt, id } = listItem
       const isLink = !!link
+      console.log("isLink", image)
+      console.log("caption", caption)
       return isLink ? (
-        <ItemLink key={id || idx} to={link}>
-          {image && <Image image={image} alt={alt || ""} />}
-          <LeftContainer>
-            <ItemTitle>{title}</ItemTitle>
+        <ChakraLink
+          key={id || idx}
+          as={Link}
+          to={link}
+          textDecoration="none"
+          display="flex"
+          justifyContent="space-between"
+          color="text"
+          marginBottom="1px"
+          padding="1rem"
+          width="100%"
+          boxShadow="0px 1px 1px var(--eth-colors-tableItemBoxShadow)"
+          _hover={{
+            textDecoration: "none",
+            boxShadow: "0 0 1px var(--eth-colors-primary)",
+            bg: "primary100",
+            color: "black",
+          }}
+        >
+          {image && (
+            <GatsbyImage
+              style={{
+                marginTop: "4px",
+                minWidth: "20px",
+                marginRight: "1rem",
```

Comment:
You can use the `Image` component from Chakra to avoid doing this. https://chakra-ui.com/docs/components/image/usage

```tsx
<Image as={GatsbyImage} mt={1} ... />
```

Reply:
Did the change!